src/core/api/chart_repository.go

```diff
@@ -554,3 +651,29 @@ func chartFullName(namespace, chartName, version string) string {
 	}
 	return fmt.Sprintf("%s/%s:%s", namespace, chartName, version)
 }
+
+// Returns true if chart of version exists in namespace
+var chartVersionExists = func(namespace, chartName, version string) bool {
```

Comment:
Why the `chartVersionExists` and `parseChart` are variables? Can't we just define the function?

Reply:
They are defined as variables so that can be mocked in chart repository test case.